Assets/HoloToolkit/Utilities/Scripts/BoundingBoxHelper.cs

```diff
@@ -0,0 +1,332 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// The BoundingBoxHelper class contains functions for getting geometric info from the non-axis-aligned 
+/// bounding box of a GameObject. These functions can be used to align another object to the center of
+/// a certain face or the center of an edge of a face... etc.
+/// The BoundingBoxHelper static function can be used for a one time calculation.
+/// The dynamic functions can be used to obtain boundingcube info on an object's Update loop. Operations
+/// are minimized in the dynamic use scenario.
+/// </summary>
+public class BoundingBoxHelper : MonoBehaviour
```

Comment:
Why isn't this class in a namespace?

Reply:
It's also in Utilities, while the rest of bounding box is in UX.

Reply:
This was merged a month ago.... Please open a new issue so we can track the needed change(s)